src/main/kotlin/app/revanced/patches/all/ads/blocklist/AndroidAdsBlocklist.kt

```diff
@@ -0,0 +1,32 @@
+package app.revanced.patches.all.ads.bytecode
+
+import org.jf.dexlib2.Opcode
+
+val blockUrls: List<String> = listOf(
+    "adservice.google.com",
+    "g.doubleclick.net",
+    "googleadservices.com",
+    "amazon-adsystem.com",
+)
+
+val replaceUrlsWith: String = "https://example.com"
```

Comment:
Would it be possible to use something like `http://127.0.0.1` to avoid the extra data traffic?

Reply:
I believe so, in the end it depends to the app. An app could crash if it can't reach a URL if it was previously sure, that it would reach it for example.

Reply:
I believe 0.0.0.0 is preferred over 127.0.0.1 because there is no timeout for `0.0.0.0`. Most `/etc/hosts`-based adblockers use it for this reason. Wouldn't receiving unexpected content be more likely to crash apps compared to not getting anything at all?